apps/demos/utils/visual-tests/matrix-test-helper.js

```diff
@@ -175,17 +175,17 @@ const SKIPPED_TESTS = {
       { demo: 'TooltipHTMLSupport', themes: [THEME.material] },
     ],
     DataGrid: [
-      { demo: 'BatchUpdateRequest', themes: [THEME.fluent, THEME.material] },
-      { demo: 'ColumnCustomization', themes: [THEME.fluent] },
       { demo: 'CellEditingAndEditingAPI', themes: [THEME.material] },
+      // It looks like we don't have a "close" button in the Fluent theme.
+      // Need need to update the behavior in Fluent or find another common way for all topics.
```

Comment:
It looks like a oversight in the Fluent theme development. It would be better to involve @Shuvaevmedia  to check the UI/UX of the DataGrid's field chooser in Fluent.

Reply:
I prefer we left this on Grids responsibility. They are already aware of necessity to fix these tests.